apps/jobs/views.py

```diff
@@ -1440,8 +1441,15 @@ def update_partially_evaluated_submission(request, challenge_pk):
         stderr_content = request.data.get("stderr", "")
         submission_result = request.data.get("result", "")
         metadata = request.data.get("metadata", "")
-        submission = get_submission_model(submission_pk)
-
+        try:
+            submission = get_submission_model(submission_pk)
+        except NotFound:
```

Comment:
@burnerlee instead of `NotFound` use `Submission.DoesNotExist`. You can see other APIs in `jobs/views.py` to see an example.

Reply:
@Ram81, I had tried this earlier. But the `get_model_object` raises an exception of the type `NotFound` and was not caught by when `Except Submission.DoesNotExist` was used

Reply:
@burnerlee @Ram81 Why are we using `get_submission_model` in a `try/catch` block? IIRC, `get_submission_model` is derived from `get_model_object` and it automatically catches the exception here -- https://github.com/Cloud-CV/EvalAI/blob/5698781b99e3fcea100430a2ef85322c0f0545db/apps/base/utils.py#L70.
Am I missing something?